### PR TITLE
chore: Docs 0.19.3

### PR DESCRIPTION
**Pull request by signorecello**
# Description

This PR adds docs at version 0.19.3 and 0.19.0.

## Problem\*

Apparently during the docs workflow refactor, we missed version 0.19.0. This PR adds it back so the docs can build correctly.

## Summary\*

- Checked out at tag 0.19.0
- Created docs **at that point in time**
- Checked out tag at 0.19.3
- Created docs **at that point in time**
- Went back to master, cherry-picked both detached commits

### CODE REVIEW

**Comment by Savio-Sou** [comment]
Thanks Ze. Curious why do we need v0.19.0? (Should we remove v0.19.2 as well and just keep v0.19.3?)

**Comment by signorecello** [comment]
The idea is that the repo holds all the versions, since we never know which one will be marked as stable. When we mark any of them as stable or pre-release, it will remove the auto-releases at that point in time 😊

File: docs/versions.json [hunk 1]
```diff
@@ -1,5 +1,9 @@
 [
+  "v0.19.3",
   "v0.19.2",
+  "v0.19.0",
+  "v0.17.0",
+  "v0.10.5",
```
Comment (Savio-Sou) [inline docs/versions.json:6]:
```suggestion
```

File: docs/versions.json [hunk 2]
```diff
@@ -1,5 +1,9 @@
 [
+  "v0.19.3",
```
Comment (Savio-Sou) [inline docs/versions.json:2]: Is the expected workflow to flip this on before it is a stable release?

**Review by Savio-Sou — APPROVED** [review]

File: docs/versions.json (hunk 2 above)
Comment (Savio-Sou) [reply docs/versions.json:2]: Will be updated once the [setStable.js](https://github.com/noir-lang/noir/blob/master/docs/scripts/setStable.js) script runs, so should be fine.

**Pull request merged**